File: NewUI/src/app/models/timeTableD.ts
import { timeTableH } from "./timeTableH";
export interface timeTableD{
    ID:number;
    masterid:number;
    DayNO:number;
    ShiftCode:string;
    isdayoff:boolean;
    Name:string;
    TimePeriod:number;
    StartDate:string;
    Morning:boolean;
    Shift:boolean;
    Driver:boolean;
    shiftdriver:boolean
}

File: Api/Models/timeTableD.cs
namespace Site4Check.Models
{
    public class timeTableD
    {
        public int ID { get; set; }
        public int DayNO { get; set; }
        public string ShiftCode { get; set; }
        public bool isdayoff { get; set; }
        public int masterid { get; set; }
        public timeTableH TimeTableh { get; set; }
    }
}
